### PR TITLE
cleaned up media selection overlay styling

### LINKED ISSUE #2650: Lot of !importants in single-media-selection.scss
| Q | A
| --- | ---
| Bug? | no
| New Feature? | no
| Sulu Version | 17de020a5b4958dcd84f2812f9a280107acbf422
| Browser Version | -

#### Actual Behavior

The "single-media-selection.scss" contains a lot of "!important"s. This is quite confusing, as "!important" is not very common in our style files. Moreover the using "!important" makes the reuse of the affected files more difficult.

#### Expected Behavior

There should not be any "!important"s in the named file.

#### Possible Solutions

The application of the styles noted with "!important" should be enforced with the use of classes.

### PR DESCRIPTION
**Pull request by nnatter**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #2650 
| Related issues/PRs | -
| License | MIT
| Documentation PR | -

#### What's in this PR?

This PR cleans up the styling of the media-selection-overlay:

- the overlay container class got renamed from `single-media-selection` to `media-selection-overlay`
- removed unnecessary `!important` tags and stylings
- removed unused file `media-selection.scss`

### CODE REVIEW

**Comment by nnatter** [comment]
@danrot: this PR should be ready for review.

**Comment by danrot** [comment]
You removed all the `!important`s, which is actually good, and as I have seen that it is also working in the ckeditor, so I am wondering a bit why they have been necessary in the first place. /cc @wachterjohannes @marcelmoosbrugger 

And you have widened the overlay to better match the columns of the masonry grid, but for the widest display that doesn't really work:

image

File: src/Sulu/Bundle/MediaBundle/Resources/public/scss/modules/media-selection-overlay.scss [hunk 1]
```diff
@@ -1,37 +1,40 @@
-// The "!important"s are due to the special behavior of the CK-Editor
-// https://github.com/sulu/sulu/issues/2650
-.single-media-selection {
+.media-selection-overlay {
     & {
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/scss/modules/media-selection-overlay.scss]: Does that make any sense?

File: src/Sulu/Bundle/MediaBundle/Resources/public/scss/modules/media-selection-overlay.scss [hunk 2]
```diff
@@ -1,37 +1,40 @@
...
-        height: 500px;
-
-        &, .slide, .overlay-footer {
-            width: 930px !important;
+        // responsive width styling
+        // we do not use the responsive skin ov the overlay component due better adaptability to changing media sizes
```
Comment (danrot) [inline src/Sulu/Bundle/MediaBundle/Resources/public/scss/modules/media-selection-overlay.scss]: `ov`?

**Pull request merged**